### PR TITLE
🔧 action list price is_null

### LINKED ISSUE #11175: Explore Items Listing Not working
### What happened?

either you are able to list and it's not working or you shouldn't be able to from this page and we should hide the "select" button

https://github.com/user-attachments/assets/2a9d7111-8e0c-458d-9674-bdf4edfe08ea

### PR DESCRIPTION
**Pull request by roiLeo**
- closes #11175 
- `price` is null, it's causing the bug since we do `price_eq`/`price_gt`: `0`

note: pain to debug

### CODE REVIEW

**Review by preschian — APPROVED** [review]

**Comment by hassnian** [comment]
@roiLeo it's working for some NFTs and for some it's still not 

https://github.com/user-attachments/assets/502a898e-d443-4502-a22a-2565892072e6

**Comment by roiLeo** [comment]
> @roiLeo it's working for some NFTs and for some it's still not

can't reproduce, could you share your address?

File: components/items/ItemsGrid/ItemsGridImageTokenEntity.vue [hunk 1]
```diff
@@ -200,9 +200,8 @@ const onClickShoppingCart = async () => {
 const onClickListingCart = async () => {
   const nftsToProcess = await getTokensNfts([props.entity])
 
-  for (const nft of nftsToProcess) {
```
Comment (roiLeo) [inline components/items/ItemsGrid/ItemsGridImageTokenEntity.vue:203]: This part make a weird loop of the same items on some generative nfts

Comment (vikiival) [reply components/items/ItemsGrid/ItemsGridImageTokenEntity.vue:203]: cc @Jarsen136 ?

Comment (roiLeo) [reply components/items/ItemsGrid/ItemsGridImageTokenEntity.vue]: thanks for the clarification, i've added missing variables

File: components/items/ItemsGrid/useNftActions.ts [hunk 2]
```diff
@@ -40,6 +40,12 @@ export const prepareListingQuery = (
         token: tokenSearchTerm,
         currentOwner_eq: accountId.value,
         burned_eq: false,
+        OR: {
+          price_isNull: true,
+          token: tokenSearchTerm,
+          currentOwner_eq: accountId.value,
+          burned_eq: false,
+        },
```
Comment (Jarsen136) [inline components/items/ItemsGrid/useNftActions.ts:48]: I'm confused about this OR condition here. Would it be possible to remove the price limit (lines 53- 54) instead?

image

Comment (roiLeo) [reply components/items/ItemsGrid/useNftActions.ts:48]: if there are no edge case Ok

**Review by Jarsen136 — APPROVED** [review]

**Pull request merged**